### ISSUE TITLE
bug: SYP service is down notice 

### ISSUE THREAD

**Issue by natalianorori** [oaworks/discussion#1827]
The SYP website, as well as Montana's version of it appear to be down. I have tried different DOIs and get this message: Oh dear, the service is down! We're aware, and working to fix the problem. Please try contacting us directly

Are we aware of this? 
@JosephMcArthur @markmacgillivray

**Comment by JosephMcArthur** [comment]
Can you share which DOIs you've tried? I can't replicate it right now.

Joseph McArthur
Assistant Director: Right to Research Coalition <http://righttoresearch.org>
Director / Founder: Open Access Button <http://openaccessbutton.org>
Twitter: @Mcarthur_Joe <https://twitter.com/Mcarthur_Joe>
Skype: joseph_mcarthur

**Comment by natalianorori** [comment]
10.1016/j.ijcard.2018.10.089
10.1093/neuros/nyab273
10.1021/acsnano.8b01396

But I just tried again and could access it.. so I guess it might have been temporal

**Comment by JosephMcArthur** [comment]
Ditto. The system can recover itself sometimes. Interestingly, the uptime
monitoring systems didn't fire. How long did see it down for during your
testing?

Joseph McArthur
Assistant Director: Right to Research Coalition <http://righttoresearch.org>
Director / Founder: Open Access Button <http://openaccessbutton.org>
Twitter: @Mcarthur_Joe <https://twitter.com/Mcarthur_Joe>
Skype: joseph_mcarthur

**Comment by natalianorori** [comment]
It was down for around 10 minutes

**Comment by JosephMcArthur** [comment]
Thanks! I should've asked, got a rough time of that ten minutes? E.g 9:29pm
to 9:39pm.

Mark, can you see if there's anything to learn from the logs next week?

Joseph McArthur
Assistant Director: Right to Research Coalition <http://righttoresearch.org>
Director / Founder: Open Access Button <http://openaccessbutton.org>
Twitter: @Mcarthur_Joe <https://twitter.com/Mcarthur_Joe>
Skype: joseph_mcarthur

**Comment by natalianorori** [comment]
9:28-9:38 might be a good time lapse

**Comment by markmacgillivray** [comment]
I reviewed the logs for that time and they seem fine, there are certainly records of finds hitting the backend. It could have been a localised issue where the service couldn't be reach across the network for only certain places. I've resurrected issue #1822 to keep track of the SYP problems as they relate to deposit, and we have a general task of needing to move that to the new API anyway. So probably not worth putting much more time into investigating this. I'll close, but re-open if you have anything else you would like me to check.
